router/build.sh

```diff
@@ -38,14 +40,11 @@ apk add --update-cache \
   zlib \
   zlib-dev
 
-# grab the source files
-curl -sSL http://nginx.org/download/$VERSION_NGINX.tar.gz -o $BUILD_PATH/$VERSION_NGINX.tar.gz
-echo "$SHA256_NGINX *$VERSION_NGINX.tar.gz" | sha256sum -c - || exit 10
-curl -sSL https://github.com/nbs-system/naxsi/archive/$VERSION_NAXSI.tar.gz -o $BUILD_PATH/$VERSION_NAXSI.tar.gz
-
-# expand the source files
-tar xzf $VERSION_NGINX.tar.gz
-tar xzf $VERSION_NAXSI.tar.gz
+# download and extract the source files
+curl -sSL http://nginx.org/download/$VERSION_NGINX.tar.gz | tar xz
+curl -sSL https://github.com/nbs-system/naxsi/archive/$VERSION_NAXSI.tar.gz | tar xz
+curl -sSL https://github.com/simpl/ngx_devel_kit/archive/v$VERSION_NDK.tar.gz | tar xz
+curl -sSL https://github.com/openresty/set-misc-nginx-module/archive/v$VERSION_SETMISC.tar.gz | tar xz
```

Comment:
I plan to rewrite this to include @bladealslayer changes on #3954 – also, to include verification of the plugins as well